lib/grpc/transport/http2.ex

```diff
@@ -33,6 +33,7 @@ defmodule GRPC.Transport.HTTP2 do
   @spec client_headers_without_reserved(GRPC.Client.Stream.t(), map) :: [{String.t(), String.t()}]
   def client_headers_without_reserved(%{codec: codec} = stream, opts \\ %{}) do
     [
+      {"authorization", "#{opts[:authorization] || ""}"},
```

Comment:
How about using `opts[:authorization]` directly?

Reply:
What about remove this line and have something like this:

`|> append_authorization(opts[:authorization])`

and then:
```
defp append_authorization(headers, authorization) when is_bitstring(authorization) do
	[{"authorization", "#{authorization}"} | headers]
end
```

This way we avoid send extra headers